lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ServiceRegistry.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import uk.gov.di.ipv.cri.common.library.persistence.DataStore;
+import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
+import uk.gov.di.ipv.cri.common.library.persistence.item.SessionItem;
+import uk.gov.di.ipv.cri.common.library.service.AuditEventFactory;
+import uk.gov.di.ipv.cri.common.library.service.AuditService;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.common.library.service.PersonIdentityService;
+import uk.gov.di.ipv.cri.common.library.service.SessionService;
+import uk.gov.di.ipv.cri.common.library.util.ListUtil;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KBVGatewayFactory;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KeyStoreLoader;
+
+import java.time.Clock;
+
+public class ServiceRegistry {
+    private final ConfigurationService configurationService;
+    private AuditService auditService;
+    private SessionService sessionService;
+    private Clock clock = Clock.systemUTC();
+    private boolean isLoaded;
+
+    public ServiceRegistry(ConfigurationService configurationService) {
+        this.configurationService = configurationService;
```

Comment:
the AuditService / SessionService can be made `final` and initialised in the constructor
the clock field can be removed as it won't be needed when the new constructors are used